packages/rest/test/unit/rest.server/rest.server.open-api-spec.unit.ts

```diff
@@ -89,6 +89,29 @@ describe('RestServer.getApiSpec()', () => {
     });
   });
 
+  it('ignores routes marked as "x-internal" via app.route(route)', () => {
+    function greet() {}
+    function meet() {}
+    server.route(
+      new Route(
+        'get',
+        '/greet',
+        {'x-internal': true, responses: {}, spec: {}},
+        greet,
+      ),
+    );
+    server.route(new Route('get', '/meet', {responses: {}, spec: {}}, meet));
```

Comment:
This check feels rather redundant to me - we already have tests that are verifying how regular (non-internal) routes are published to the spec, no need to duplicate those test again.

In case you convince me otherwise, then replace `server.route(new Route(...))` to `server.route(...)`.

Reply:
In my first bad attempt some days ago, setting `x-internal` hid all the routes. Let's ensure `x-internal` does not have side effects.